modules/oadp-creating-backup-cr.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-creating-backup-cr_{context}"]
+= Creating a Backup CR
+
+You back up Kubernetes images, internal images, and persistent volumes (PVs) by creating a `Backup` custom resource (CR).
+
+.Prerequisites
+
+* Backup prerequisites:
+** You must have S3 object storage configured for Velero.
+** You must have a backup location configured in the `DataProtectionApplication` CR.
+* Snapshot prerequisites:
+** You must have a cloud provider that supports snapshots or the Cloud Snapshot Interface (CSI).
```

Comment:
>the Cloud Snapshot Interface (CSI).

1. CSI = `Container Snapshot Interface`
2. It seems the description doesn't consider using CSI driver for OCS 4 snapshot

Reply:
other looks good to me  @apinnick

Reply:
CSI - good catch. I will check all the other PRs to make sure I didn't do the same typo.

Reply:
I added a reference to the CSI driver and OCS 4